### ISSUE TITLE
CLI config  issue

### ISSUE THREAD

**Issue by jch2711** [databricks/databricks-sdk-go#891]
**Description**
Installed CLI on Ubantu machine. Installation successful and configure successful using token and url. But when trying to list available cluster getting error.

**Reproduction**
databricks clusters list

Error: unexpected error handling request: invalid character '<' looking for beginning of value. This is likely a bug in the Databricks SDK for Go or the underlying REST API. Please report this issue with the following debugging information to the SDK issue tracker at https://github.com/databricks/databricks-sdk-go/issues. Request log:

```

GET /login.html?error=private-link-validation-error:7034804820594289

> * Host: 

> * Accept: application/json

> * Authorization: REDACTED

> * Referer: https://7034804820594289.9.gcp.databricks.com/api/2.0/clusters/list?

> * User-Agent: cli/0.217.1 databricks-sdk-go/0.37.0 go/1.21.9 os/linux cmd/clusters_list sdk-feature/pagination auth/pat

< HTTP/2.0 200 OK

< * Cache-Control: no-cache, no-store, must-revalidate

< * Content-Security-Policy: default-src *; font-src * data:; frame-src * blob:; img-src * blob: data:; media-src * data:; object-src 'none'; style-src * 'unsafe-inline'; worker-src * blob:; script-src 'self' 'unsafe-eval' 'unsafe-hashes' 'report-sample' https://*.databricks.com https://databricks.github.io/debug-bookmarklet/ https://widget.intercom.io https://js.intercomcdn.com https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js https://databricks-ui-assets.azureedge.net https://ui-serving-cdn-testing.azureedge.net https://uiserviceprodwestus-cdn-endpoint.azureedge.net https://databricks-ui-infra.s3.us-west-2.amazonaws.com 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=' 'sha256-psD2qcIM2XnZVqZI24l9uDm/mj3q5XG44VaKlJf3CRc=' 'sha256-Lh4yp7cr3YOJ3MOn6erNz3E3WI0JA20mWV+0RuuviFM=' 'sha256-nzRwRRdIksZqnL5EOMg/DYbemU4fIcFgm7ig5Nz3SiA='; report-uri /ui-csp-reports; frame-ancestors *.vocareum.com *.docebosaas.com *.edx.org *.deloitte.com *.cloudlabs.ai *.databricks.com

< * Content-Type: text/html; charset=utf-8

< * Date: Fri, 12 Apr 2024 19:02:39 GMT

< * Server: databricks

< * Strict-Transport-Security: max-age=31536000; includeSubDomains; preload

< * X-Content-Type-Options: nosniff

< * X-Ui-Svc: true

< * X-Xss-Protection: 1; mode=block

< <!doctype html>

< <html>

<  <head>

<   <meta charset="utf-8">

<   <meta http-equiv="Content-Language" content="en">

<   <title>Databricks - Sign In</title>

<   <meta name="viewport" content="width=960">

<   <link rel="icon" type="image/png" href="https://ui-assets.gcp.databricks.com/favicon.ico">

<   <meta http-equiv="content-type" content="text/html; charset=UTF8">

<   <script id="__databricks_react_script"></script>

<   <script>window.__DATABRICKS_SAFE_FLAGS__={"databricks.fe.infra.fixConfigPrefetch":true,"databricks.fe.infra.useReact18":true,"databricks.infra.showErrorModalOnFetchError":true,"databricks.fe.infra.useReact18NewAPI":false,"databricks.fe.infra.useNewRnDTaxonomy":false},window.__DATABRICKS_CONFIG__={"publicPath":{"mlflow":"https://ui-assets.gcp.databricks.com/","dbsql":"https://ui-assets.gcp.databricks.com/","feature-store":"https://ui-assets.gcp.databricks.com/","monolith":"https://ui-assets.gcp.databricks.com/","jaws":"https://ui-assets.gcp.databricks.com/"}}</script>

<   <link rel="icon" href="https://ui-assets.gcp.databricks.com/favicon.ico">

<   <script>

<   function setNoCdnAndReload() {

<       document.cookie = `x-databricks-cdn-inaccessible=true; path=/; max-age=86400`;

<       const metric = 'cdnFallbackOccurred';

<       const browserUserAgent = navigator.userAgent;

<       const browserTabId = window.browserTabId;

<       const performanceEntry = performance.getEntriesByType('resource').filter(e => e.initiatorType === 'script').slice(-1)[0]

<       sessionStorage.setItem('databricks-cdn-fallback-telemetry-key', JSON.stringify({ tags: { browserUserAgent, browserTabId }, performanceEntry}));

<       window.location.reload();

<   }

< </script>

<   <script>

<   // Set a manual timeout for dropped packets to CDN

<   function loadScriptWithTimeout(src, timeout) {

<      return new Promise((resolve, reject) => {

<         const script = document.createElement('script');

<           script.defer = true;

<           script.src = src;

<           script.onload = resolve;

<           script.onerror = reject;

<           document.head.appendChild(script);

<           setTimeout(() => {

<               reject(new Error('Script load timeout'));

<           }, timeout);

<       });

<   }

<   loadScriptWithTimeout('https://ui-assets.gcp.databricks.com/static/js/login/login.7d906ada.js', 10000).catch(setNoCdnAndReload);

< </script>

<  </head>

<  <body class="light-mode">

<   <uses-legacy-bootstrap>

<    <div id="login-page"></div>

<   </uses-legacy-bootstrap>

<  </body>

< </html>



**Expected behavior**
A clear and concise description of what you expected to happen.

**Is it a regression?**
Did this work in a previous version of the SDK? If so, which versions did you try?

**Debug Logs**
The SDK logs helpful debugging information when debug logging is enabled. Set the log level to Trace by configuring the default logger to log at trace (for example: add `logger.DefaultLogger = &logger.SimpleLogger{Level: logger.LevelTrace}` to your program), and include the logs here.

**Other Information**
 - OS: Ubantu
Databricks CLI v0.217.1


**Additional context**
Add any other context about the problem here.
